### ISSUE TITLE
Error on angle bracket within HEREDOC when declaration uses `|| true`

### ISSUE THREAD

**Issue by mattalexx** [BashSupport-Pro/bashsupport-pro#96]
Here's an example:

```bash
#!/usr/bin/env bash

_usage() {
    IFS='' read -r -d '' usagetext <<ENDUSAGETEXT || true
usage: ${0##*/} <arg>          <--------- BashSupport says "<redirection target> expected"
ENDUSAGETEXT
    printf '%s' "${usagetext}"
    exit "${1}"
}

# ...

- IntelliJ IDEA 2021.3 (Ultimate Edition)
- pro.bashsupport (2.0.9.213)

**Comment by jansorg** [comment]
Thanks! That's a problem with the parsing of `read`, I'll fix that for the next update.

**Comment by jansorg** [comment]
I've fixed this now, it'll be part of version 2.1.
